crates/emitter/src/ast_emitter.rs

```diff
@@ -2,10 +2,12 @@
 //!
 //! Converts AST nodes to bytecode.
 
-use super::emitter::{BytecodeOffset, EmitError, EmitOptions, EmitResult, InstructionWriter};
+use super::emitter::{EmitError, EmitOptions, EmitResult, InstructionWriter};
 use super::opcode::Opcode;
 use ast::types::*;
 
+use crate::forward_jump_emitter::{ForwardJumpEmitter, JumpKind, JumpPatchEmitter};
```

Comment:
I think, you don't have to import `JumpPatchEmitter`, after removing the type definition in the assignment.